controllers/articlesController.js

```diff
@@ -2,11 +2,11 @@ const {fetchArticles} = require('../models/articlesModel')
 
 const getArticles = async(req,res,next) =>{
 
-    console.log("Hi")
+   // console.log("Hi")
 
     try{
         const Articles = await fetchArticles();
-        res.status(200).json(Articles)
+        res.status(200).send(Articles)
```

Comment:
You shouldn't use a capital letter for your variables, as this is usually reserved for constructor functions or classes - regular variables should be lower case